### ISSUE TITLE
Prevent blinding white screen

### ISSUE THREAD

**Issue by ThreeDeeJay** [KieronQuinn/DarQ#94]
There's a brief flash of a retina-burning white screen when opening some apps at night, and I'd very much love to Thanos-snap it out of existence. IIRC it may happen even in apps where DarQ is darkening the UI as usual.

There's *got* to be a place where #FFFFFF is set as the default background, right? Maybe I could edit that with root but I can't find any info on that.
Or is that just the default "empty" color?

**Comment by Bakr-Ali** [comment]
@ThreeDeeJay did you find any solution (hopefully non-root) for this?

**Comment by ThreeDeeJay** [comment]
Nope. I even tried enabling DarQ (list and LSPosed scope) for Trebuchet (stock launcher) and System UI but no luck even after a restart. Perhaps it just hooks on the app too late. @KieronQuinn can you confirm?
